### DISCUSSION TITLE
Hidden amount.

### DISCUSSION THREAD

**Discussion by shinebrillant** [enrique-lozano/Monekin#173] [Ideas / New features]
Hide the amount in the software and switch between showing and hiding it after clicking to increase privacy.

**Comment by enrique-lozano** [comment]
I honestly think this is a great idea that could differentiate Monekin from other similar apps, as not all of them have this functionality. Among those that do, I think a very good example is how Wallet (by BudgetBakers) does it, through a toggle in the left drawer. 

Taking into account that we should hide all the amounts to increase the privacy and considering the current design, I thought of the following:

- Create a toggle option inside some of the settings pages to toggle what we will call a "private mode". This toggle will be like the one to toggle the amoled mode.
- Since we will need at least 3-4 clicks to change this toggle (in Wallet for example we need only two), we will need some kind of "shortcut" to toggle this mode. I think that a long press in the total balance of the dashboard wil be enough.
- We could not toggle this mode in each click of every amount in the app, since some amounts are inside other clicking events and it will be very very difficult and confusing to implement.

Please, ket me know what you think about this proposal. Any new ideas are welcome.

Thanks a lot!

**Comment by shinebrillant** [comment]
The way I can think of ~~excessive demands~~ is to switch the privacy mode by long pressing or clicking the Dashboard at the bottom of the screen multiple times, hiding the part about the account balance, retaining the percentage increase or decrease, and hiding the Y-axis amount in the chart part.
Hide amounts in Transactions.

**Reply by enrique-lozano** [reply]
Yep, I agree. This is in line with what I said in the previous post.

I will move this to the issues section as it will be 100% implemented this way in the future.

**Comment by enrique-lozano** [comment]
Implemented in #188
